cmake/linker/arcmwdt/target.cmake

```diff
@@ -97,9 +97,9 @@ function(toolchain_ld_link_elf)
     ${LINKERFLAGPREFIX}--entry=__start
     ${LINKERFLAGPREFIX}--Map=${TOOLCHAIN_LD_LINK_ELF_OUTPUT_MAP}
     ${LINKERFLAGPREFIX}--whole-archive
-    ${ZEPHYR_LIBS_PROPERTY}
-    ${LINKERFLAGPREFIX}--no-whole-archive
-    kernel
+    ${WHOLE_ARCHIVE_LIBS}
+    ${LINKERFLAGPREFIX},--no-whole-archive
```

Comment:
Tthis linker, unlike all others, does not seem to use commas... :man_shrugging: is this a typo?

Reply:
Oh noes, copypasta detected!